src/test/java/jssc/bootpath/Helper.java

```diff
@@ -0,0 +1,20 @@
+package jssc.bootpath;
+
+import org.scijava.nativelib.NativeLibraryUtil;
+
+/**
+ * TODO: Invocations MUST be in their own class to run in a separate JVM (https://stackoverflow.com/questions/68657855)
+ * - JUnit does NOT currently offer JVM unloading between methods.
+ * - maven-surefire-plugin DOES offer JVM unloading between classes using <code>reuseForks=false</code>
+ * - Unloading is needed due to NativeLoader.loadLibrary(...) calls System.loadLibrary(...) which is static
+ * */
+public class Helper {
+    public static void fixNativesPath() {
+        String nativeLibDir = NativeLibraryUtil.getPlatformLibraryPath(System.getProperty("user.dir") + "/target/cmake/natives/");
+        System.setProperty("jssc.boot.library.path", nativeLibDir);
+    }
```

Comment:
We could simply set that property globally via surefire plugin properties:
```
        <groupId>org.apache.maven.plugins</groupId>
        <artifactId>maven-surefire-plugin</artifactId>
        <version>${plugin.surfire.version}</version>
        <configuration>
          <systemPropertyVariables>
            <jssc.boot.library.path>${project.build.directory}/cmake/natives/</jssc.boot.library.path>
          </systemPropertyVariables>
          <reuseForks>false</reuseForks>
          <excludes>
            <!--suppress UnresolvedMavenProperty -->
            <exclude>${maven.exclude.tests}</exclude>
          </excludes>
        </configuration>
```
The test that needs to change it does it explicity, but all other are automatically handled

Reply:
Good idea! I'll rework this.

Reply:
Reverted due to inability to detect `.dll`, `.so`, `.dylib` path.  We'll continue to use the old method for now.